### ISSUE TITLE
Hook Wii console btn reset as a key for mapping commands

### ISSUE THREAD

**Issue by SuperrSonic** [libretro/RetroArch#757]
Personally there are a few ways to call RGUI and I found that it was more convenient if the reset button would reset the game instead of calling the menu but instead of forcing this change to every user some users might not like it, in this case it would be better if this option was customizable and we can already map to "reset content" but for controllers only.

In gx_input.c
static void reset_cb(void)
{
   g_menu = true;
}
I tried manually doing it but failed, tried making a "g_reset" but I have no idea how to connect it with RARCH_RESET the way g_menu is to RARCH_MENU_TOGGLE

**Comment by twinaphex** [comment]
I agree we should finally look into getting this done. PRs would be welcome though.
